civictechprojects/static/css/partials/_MainController.scss

```diff
@@ -1,3 +1,2 @@
 #MainController {
```

Comment:
We can remove this rule (and the file, in fact) if it's not doing anything anymore.